CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Features ✨ and improvements 🏁
+* Add `Widget.triggerRepaint` protected API and updated the `BitmapWidget` to trigger repaint after widget is updated. ([1797](https://github.com/mapbox/mapbox-maps-android/pull/1797))
```

Comment:
@pengdev I think technically we could still land this in RC, wdyt? see no major risks, this API is nice addition

Reply:
There's one potential public API addition, but it's only visible for classes that extends Widget(where the constructor is internal). I think the public API surface is minimal, I'd agree to include in RC. (given the widget API is still experimental)

Reply:
if we decide to do so - we could already create v10.10.0-rc.1 section in changelog then as `#main` indicates it will be available in next minor release

Reply:
Btw - do we need to mention the internal API? Maybe just say that `update the widget when a property changes` ?

Reply:
As you can see there's changes in the api file, so definitely there's API changes, however given the fact that user can not actually use the new APIs(only extendable for internal classes), maybe I can just make it internal, so there wouldn't be additions in the API file.

Updated, please take another look.